content/projects/gsoc/2020/project-ideas/machine-learning.adoc

```diff
@@ -4,16 +4,15 @@ title: "Jenkins and Machine Learning Plugins for Data Science"
 goal: "Create a new plugin for integrating Jenkins with one of Machine Learning tools (e.g. Jupyter Python, TensorBoard, or Sacred)"
 category: Plugins
 year: 2020
-status: draft 
-showGoogleDoc: true
+status: published
+sig: pipeline authoring
 skills:
 - Java
```

Comment:
Not a blocker, but labels have not been moved yet

Reply:
Still not sure what is needed here?

Reply:
Labels you have in Google Doc?

Reply:
https://docs.google.com/document/d/19ignQBMUr3qxfmkf8Sa9KG7wJlxs3js_kg4mJhX_dXo/edit#heading=h.2677a7f3650i